docs/document_loader.ipynb

```diff
@@ -119,23 +105,62 @@
    }
   },
   {
-   "cell_type": "code",
+   "cell_type": "markdown",
    "source": [
-    "from langchain_google_el_carro import ElCarroEngine\n",
     "\n",
-    "elcarro_engine = ElCarroEngine.from_instance(\n",
-    "    db_host=HOST,\n",
-    "    db_port=PORT,\n",
-    "    db_name=DATABASE,\n",
-    "    db_user=USER,\n",
-    "    db_password=PASSWORD,\n",
-    ")"
+    "If you are using El Carro, you can find the hostname and port values in the\n",
+    "status of the El Carro Kubernetes instance.\n",
+    "Use the user password you created for your PDB.\n",
+    "\n",
+    "Example Ouput:\n"
    ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+     "```\n",
+    "kubectl get -w instances.oracle.db.anthosapis.com -n db\n",
+    "NAME   DB ENGINE   VERSION   EDITION      ENDPOINT      URL                DB NAMES   BACKUP ID   READYSTATUS   READYREASON        DBREADYSTATUS   DBREADYREASON\n",
+    "\n",
+    "mydb   Oracle      18c       Express      mydb-svc.db   34.71.69.25:6021   ['pdbname']            TRUE          CreateComplete     True            CreateComplete\n",
```

Comment:
I'm not sure if this is easy to read

Reply:
hm, how would you recommend formatting this?